jll/N/NetCDF_jll/Compat.toml

```
@@ -62,6 +62,9 @@ TOML = "1"
 ["400.902.211-401"]
 MPIPreferences = "0.1"
 
+["400.902-400"]
```

Just flying by here, but wouldn't the deps entry actually imply `["400.902-401"]`?
```suggestion
["400.902-401"]
```